crates/sui-core/src/authority_active/execution_driver/mod.rs

```diff
@@ -242,6 +280,15 @@ where
     let executed_count = executed.len();
     debug!(?pending_count, ?executed_count, "execute_pending completed");
 
+    active_authority
+        .execution_driver_metrics
+        .pending_transactions
```

Comment:
Not sure if this is totally right. The sequential execution loop above can fail for some elements too. Maybe just set `pending_transactions` earlier from the result of `get_pending_digests()`?

Reply:
sounds good